src/CONST.ts

```diff
@@ -1705,7 +1705,7 @@ const CONST = {
         UNKNOWN_ERROR: 'Unknown error',
         REQUEST_CANCELLED: 'AbortError',
         FAILED_TO_FETCH: 'Failed to fetch',
-        ENSURE_BUGBOT: 'ENSURE_BUGBOT',
+        ENSURE_BUG_BOT: 'ENSURE_BUG_BOT',
```

Comment:
```suggestion
        ENSURE_BUGBOT: 'ENSURE_BUGBOT',
```

This needs to be `ENSURE_BUGBOT` as that is what our backend uses to track these errors.